### ISSUE TITLE
How to use .predict() function 

### ISSUE THREAD

**Issue by yafsin** [huggingface/setfit#260]
Hi,

I am new at using the setfit. I will be running many tunings for models and currently achieved getting evaluation metrics using ("trainer.evaluate())
However, is there any way to do something like the following to save the trained model's predictions?  

trainer = SetFitTrainer(......)
trainer.train()
**predictions=trainer.predict(testX)**
SetFitTrainer has no predict function

I can achieve that with trainer.push_to_hub() and downloading back with SetFitModel.from_pretrained() but there is probably a better way without publishing on the hub?

**Comment by zachschillaci27** [comment]
Hello @yafsin,

The `SetFitTrainer` is trainer class which trains an underlying `SetFitModel`. You are most likely constructing the `SetFitTrainer` with something like:
```
trainer = SetFitTrainer(
    model=model,
    train_dataset=df_train,
    ...
)
```
Here the model that you pass to the trainer is precisely the underlying `SetFit` model. That means once training is complete you can simply call `model.predict()` as follows:
```
predictions = model.predict(testX)
```
Hope this help!

**Comment by tomaarsen** [comment]
@zachschillaci27 is exactly right! I think this should be answered now, so I'll close this. If you have any other questions, then you're always free to respond, reopen the issue, or open another issue.
